connector-packager/tests/test_xsd_validator.py

```diff
@@ -119,3 +119,14 @@ def test_validate_duplicate_fields_absent(self):
         logging.debug("test_validate_duplicate_fields_absent xml violations:")
         for violation in xml_violations_buffer:
             logging.debug(violation)
+
+    def test_warn_defaultSQLDialect_as_base(self):
+
+        test_dialect_file = TEST_FOLDER / "defaultSQLDialect_as_base/dialect.tdd"
+        file_to_test = ConnectorFile("dialect.tdd", "dialect")
+
+        with self.assertLogs('packager_logger', level='WARNING') as cm:
+            check_file_content(file_to_test, test_dialect_file)
+
+        self.assertIn('WARNING:packager_logger:' + DEFAULTSQLDIALECT_AS_BASE_WARNING_MESSAGE, cm.output)
```

Comment:
Maybe just assert you get a warning and it contains "DefaultSQLDialect", that was we don't need to do a shared variable for every message going forward.